### PR TITLE
Proof Read Installation

### PR DESCRIPTION
**Pull request by eliasjpr**

### CODE REVIEW

File: guides/installation.md [hunk 1]
```diff
@@ -42,6 +43,8 @@ sudo swapon /swapfile
 * These are necessary to compile the CLI:
 * `sudo apt-get install build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev libyaml-dev`
 
+> You can also install amber using [Linuxbrew](http://linuxbrew.sh/) `brew install amber`
```
Comment (faustinoaq) [inline guides/installation.md]: Interesting, we should try [Nix](https://nixos.org/nix/) as well. I remember @manveru mentioned it on Mint channel ( [Mint is using it](https://www.mint-lang.com/install) )

Comment (eliasjpr) [reply guides/installation.md]: I guess we can create Debian packages instead using the new travis deploy? Not sure if we want to support many different package managers

**Comment by eliasjpr** [comment]
Mint is turning to be a nice project. Maybe we should make it the default front end language for amber :)

**Comment by eliasjpr** [comment]
Is not hard to create a Debian package after the binary is built http://www.king-foo.com/2011/11/creating-debianubuntu-deb-packages/

**Pull request merged**